test/suite.go

```
@@ -58,41 +63,46 @@ func (s *ExecutorSuite) skipIfInjectorNotSet() {
-	initialHeight := uint64(1)
-	chainID := "test-chain"
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), maxTestDuration)
 	defer cancel()
-
-	stateRoot, maxBytes, err := s.Exec.InitChain(ctx, genesisTime, initialHeight, chainID)
-	s.Require().NoError(err)
-	s.NotEqual(types.Hash{}, stateRoot)
-	s.Greater(maxBytes, uint64(0))
+	initialHeight := uint64(1)
+	genesisTime, stateRoot, maxBytes := s.initChain(ctx, initialHeight)
```

golangci-lint (1.62.2)

ineffectual assignment to maxBytes

(ineffassign)